fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayTabLayouts.kt

```diff
@@ -119,6 +144,8 @@ private fun TabGrid(
     onTabMediaClick: (TabSessionState) -> Unit,
     onTabClick: (TabSessionState) -> Unit,
     onTabLongClick: (TabSessionState) -> Unit,
+    onMove: (String, String?, Boolean) -> Unit,
+    onExitSelectionMode: () -> Unit,
```

Comment:
should we update this as? 
```suggestion
    onTabDragStart: () -> Unit,
```